From: SeongJae Park <sj@kernel.org>
Date: Fri, 9 Jan 2026 17:54:08 -0800
Subject: [PATCH] Docs/mm/damon/maintainer-profile: do not suggest running
 performance DAMON tests

The test has been broken for a while, and no active plan to fix it
exists for now.  Do not suggest running it for now, as any attempt to
run it should be only waste of time for now.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/mm/damon/maintainer-profile.rst | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Documentation/mm/damon/maintainer-profile.rst b/Documentation/mm/damon/maintainer-profile.rst
index 4fa06b21ec30..82fa0509473c 100644
--- a/Documentation/mm/damon/maintainer-profile.rst
+++ b/Documentation/mm/damon/maintainer-profile.rst
@@ -48,9 +48,6 @@ Further doing below and putting the results will be helpful.
 - Run `damon-tests/corr
   <https://github.com/damonitor/damon-tests/tree/master/corr>`_ for normal
   changes.
-- Run `damon-tests/perf
-  <https://github.com/damonitor/damon-tests/tree/master/perf>`_ for performance
-  changes.
 
 Key cycle dates
 ---------------
-- 
2.47.3

